packages/core/config/fileMock.js

```diff
@@ -0,0 +1 @@
+module.exports = 'fileMock'
```

Comment:
Do you think this file could be part of `@redwoodjs/testing`

Reply:
I think it could. Do you think it would be a good idea to move other testing stuff like Jest config and the setup files to that `@redwoodjs/testing` as well? And have the testing package sort of dogfood the rest of the monorepo?

Reply:
I would like to keep configuration in core. In my mind the "core" package is what a user's requires to develop and build a redwood project and the configurations are part of that.

The file mock feels like something that should belong to test, whereas this configuration is something that's using "@redwoodjs/testing" - that distinction may seem a bit loosey-goosey and I'll try to clarify it with better language.